frontend/src/app/AdminNavBar/AdminNavBar.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import { BiCommentDetail } from 'react-icons/bi'
+import { Link as ReactLink } from 'react-router-dom'
+import {
+  As,
+  chakra,
+  Flex,
+  FlexProps,
+  HStack,
+  useDisclosure,
+} from '@chakra-ui/react'
+
+import { BxsHelpCircle } from '~assets/icons/BxsHelpCircle'
+import { ReactComponent as BrandMarkSvg } from '~assets/svgs/brand/brand-mark-colour.svg'
+import { useIsMobile } from '~hooks/useIsMobile'
+import { logout } from '~services/AuthService'
+import IconButton from '~components/IconButton'
+import Link from '~components/Link'
+import { AvatarMenu, AvatarMenuDivider } from '~templates/AvatarMenu/AvatarMenu'
+
+import { EmergencyContactModal } from '~features/user/emergency-contact/EmergencyContactModal'
+import { useUser } from '~features/user/queries'
+
+import Menu from '../../components/Menu'
+
+const BrandSmallLogo = chakra(BrandMarkSvg)
+
+const NAV_LINKS = [
+  {
+    label: 'Feature request',
+    href: '',
+    MobileIcon: BiCommentDetail,
+  },
+  {
+    label: 'Help',
+    href: 'https://guide.form.gov.sg',
+    MobileIcon: BxsHelpCircle,
+  },
+]
+
+type AdminNavBarLinkProps = {
+  label: string
+  href: string
+  MobileIcon?: As
+}
+
+const AdminNavBarLink = ({ MobileIcon, href, label }: AdminNavBarLinkProps) => {
+  const isMobile = useIsMobile()
+
+  if (isMobile && MobileIcon) {
+    return (
+      <IconButton
+        variant="clear"
+        as="a"
+        href={href}
+        aria-label={label}
+        icon={<MobileIcon fontSize="1.25rem" color="primary.500" />}
+      />
+    )
+  }
+
+  return (
+    <Link
+      w="fit-content"
+      variant="standalone"
+      color="secondary.500"
+      href={href}
+      aria-label={label}
+    >
+      {label}
+    </Link>
+  )
+}
+
+export interface AdminNavBarProps {
+  /* This prop is only for testing to show expanded menu state */
+  isMenuOpen?: boolean
+}
+
+export const AdminNavBar = ({ isMenuOpen }: AdminNavBarProps): JSX.Element => {
+  const { user } = useUser()
+
+  const {
+    isOpen: isContactModalOpen,
+    onClose: onContactModalClose,
+    onOpen: onContactModalOpen,
+  } = useDisclosure()
+
+  return (
+    <>
+      <AdminNavBar.Container>
+        <Link title="Form Logo" href="https://form.gov.sg/">
+          {<BrandSmallLogo w="2rem" />}
+        </Link>
+        <HStack
+          textStyle="subhead-1"
+          spacing={{ base: '0.75rem', md: '1.5rem' }}
+        >
+          {NAV_LINKS?.map((link, index) => (
```

Comment:
/nit: since `NAV_LINKS` is a module-declared constant, it cannot be `undefined`, and so the conditional property access is not necessary.
```suggestion
          {NAV_LINKS.map((link, index) => (
```